isodesign/base/process.py

```diff
@@ -230,6 +230,12 @@ def add_isotopomer(self, substrate_name, labelling, nb_intervals, lower_b, upper
         """
 
         isotopomer = Isotopomer(substrate_name, labelling, nb_intervals, lower_b, upper_b, price)
+        
+        # Check if the labelling length is equal to the number of carbons in the substrate
+        # by using the netan dictionary (which contains the number of carbons for each substrate)
```

Comment:
"by using clen (carbon length) from the netan (network analysis) dictionary" c'est plus clair!